aten/src/ATen/core/TensorImpl.h

```diff
@@ -1486,17 +1471,29 @@ struct CAFFE2_API TensorImpl : public c10::intrusive_ptr_target {
 //    strong refcount           TODO: pack these into one word
```

Comment:
The comment above here: "We're currently comfortably under this number;
 // let's keep it that way.  (One currently approved pending size	 // let's keep it that way." Is out of date now.